### PR TITLE
6895 - Add Payara Release Note

### LINKED ISSUE #6895: Payara Release Note
We should add a file to doc/release-notes with some information for the release notes about Payara. 

I'll use this when I'm writing the release notes for Dataverse 5.

### PR DESCRIPTION
**Pull request by djbrooke**
**What this PR does / why we need it**:

Upgrading to Payara was a big change and should get a mention in the release notes for Dataverse 5.

**Which issue(s) this PR closes**:

Closes #6895 

**Special notes for your reviewer**:

No, but suggestions/updates are welcome. 

**Suggestions on how to test this**:

No testing needed IMHO. :) 

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

No.

**Is there a release notes update needed for this change?**:

This is the release note change.

**Additional documentation**:

No.

### CODE REVIEW

File: doc/release-notes/6895-payara-release-note [hunk 1]
```diff
@@ -0,0 +1,5 @@
+##Payara 5
+
+A major upgrade of the application server will provide security updates, access to new features like MicroProfile Config API, and will enable upgrades to other core technology, such as Jakarta EE 8.
```
Comment (pdurbin) [inline doc/release-notes/6895-payara-release-note:3]: We've already switched to Jakarta EE 8 in pull request #6818.

Comment (djbrooke) [reply doc/release-notes/6895-payara-release-note:3]: @pdurbin Nice, copied this from a previous issue description. I'll make it more general.

Comment (djbrooke) [reply doc/release-notes/6895-payara-release-note]: updated in https://github.com/IQSS/dataverse/pull/6944/commits/7fa2c71ee37ca5ce9dffd4b640858a134f65e53d

**Comment by pdurbin** [comment]
I clicked "approve" but I'm still wondering...

If I'm on Dataverse 4.20 and I want to upgrade to Dataverse 5, where do I find instructions?

**Comment by djbrooke** [comment]
@pdurbin I handle those instructions as part of the general release note writing, aggressively copying from previous releases. This one may require a bit less copying and more original content.

**Pull request merged**